_redirects

```diff
@@ -0,0 +1,4 @@
+https://coredns.io/* https://about.coredns.io/:splat 301!
```

Comment:
there is no `about` yet. don't we need that first?

Reply:
Partially. Added on the netlify side and waiting for the final DNS config from Netlify aka IPs for the apex as well as the CNAMEs so the change only needs to happen once.